### ISSUE TITLE
CoreUI 3.4 - sidebar dropdown menus not showing if "c-sidebar-minimized" is set

### ISSUE THREAD

**Issue by AxiakJ** [coreui/coreui-free-bootstrap-admin-template#551]
Adding class "c-sidebar-minimized" as a default setting with c-sidebar causes drop down menus to not show. Expanding and Collapsing Sidebar makes drop down menus to work as expected.

### Code Example: 
`<div class="c-sidebar c-sidebar-dark c-sidebar-fixed c-sidebar-lg-show c-sidebar-minimized" id="sidebar">`

### Image Example:
![c-sidebar-minimized](https://user-images.githubusercontent.com/56797040/101260233-bf279e00-372e-11eb-9ec7-589a93cf4963.png)

**Comment by fklee** [comment]
Exact, I thought the effect should be the same as coreui 3. But in version 4, you have to hover sidemenu, then click on element to view second level menus items, which is less friendly. The effect is the same in the demo page.

**Comment by thesea2201** [comment]
Same here, always add the class name `"ps"` after `"c-sidebar-nav"` when initialize. When I hover the sidebar button, expanding does not work as intended.
![image](https://user-images.githubusercontent.com/21289697/129478989-f0a8dbb4-0f0a-43f7-a7b5-2f1bd57c2ab7.png)

**Comment by Rasoul26** [comment]
Hi thesea2201

If you use the following code, the problem may be solved:

```
@media (min-width: 992px) {
	.c-sidebar-minimized .c-sidebar-nav {
		overflow: visible !important;
	}
}
```

Good Luck 👍
